src/autocomplete.css

```diff
@@ -131,6 +131,19 @@
   padding: 5px;
 }
 
+.autocomplete__option--ios-patch {
```

Comment:
I think we'd want to call this class name `.autocomplete__visually-hidden` to match GOV.UK Design Systems' `govuk-visually-hidden`